public/scripts/editor/js/fc/filesystem-sync.js

```diff
@@ -28,72 +28,120 @@ define(function(require) {
       processData: false
     };
 
-    function send() {
+    path = Project.stripRoot(path);
```

Comment:
This should not happen here because we do a `readFile` below which will use this path (which does not exist)